src/test/kotlin/no/nav/familie/ef/søknad/integrationTest/OppslagControllerIntegrationTest.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.familie.ef.søknad.integrationTest
+
+
+import no.nav.familie.ef.søknad.ApplicationLocalLauncher
+import no.nav.security.token.support.core.JwtTokenConstants
+import no.nav.security.token.support.test.JwtTokenGenerator
+import org.assertj.core.api.Assertions.assertThat
+import org.glassfish.jersey.logging.LoggingFeature
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import javax.ws.rs.client.ClientBuilder
+
+
+@ActiveProfiles("local", "mock-pdl", "mock-pdlStsClient")
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = [ApplicationLocalLauncher::class])
+class OppslagControllerIntegrationTest {
+
+    @Value("\${local.server.port}")
+    val port: Int = 0
+    val contextPath = "/api/oppslag"
+    val tokenSubject = "12345678911"
+
```

Comment:
Her tror jeg det hadde vært veldig fint å ha flere tester, som verifiserer mot forvented JSON og ikke kun att _navn ikke er med.

1. test som viser hva som blir returnert hvis forelder, barn og annen forelder har adressebeskyttelse
2. Test som viser hva som blir returnert hvis ingen har adressebeskyttelse
3. Test som håndterer caset når ikke søker, men barn/annen forelder har adressebeskyttelse (her blir det kastet feil)

Er det riktig att vi skal returnere FNR til barn/annen forelder hvis de er kode 6/7? For case 1 er dataen som returneres
```json
{
  "søker": {
    "fnr": "12345678911",
    "forkortetNavn": "Fornavn mellomnavn Etternavn",
    "adresse": {
      "adresse": "Charlies vei 13 b",
      "postnummer": "0575",
      "poststed": ""
    },
    "egenansatt": false,
    "sivilstand": "SKILT",
    "statsborgerskap": ", ",
    "harAdressesperre": true
  },
  "barn": [
    {
      "fnr": "28021078036",
      "alder": 0,
      "fødselsdato": "2020-11-03",
      "harSammeAdresse": true,
      "medforelder": {
        "harAdressesperre": true,
        "død": false,
        "ident": "01049940234",
        "navn": "",
        "alder": 22
      },
      "harAdressesperre": true,
      "navn": ""
    }
  ],
  "hash": "841480147"
}
```